pom.xml

```diff
@@ -285,6 +285,12 @@
             <artifactId>jcommon</artifactId>
             <version>1.0.24</version>
         </dependency>
```

Comment:
Now that we have WebP, we can get rid of the `jcommon` lib and its PNG encoder.

Reply:
Done